content/docs/2.0/scalers/ibm-mq.md

```diff
@@ -0,0 +1,96 @@
+> © Copyright IBM Corporation 2020
+> 
+> Licensed under the Apache License, Version 2.0 (the "License"); you
+> may not use this file except in compliance with the License. You may
+> obtain a copy of the License at
+> 
+> http://www.apache.org/licenses/LICENSE-2.0
+> 
+> Unless required by applicable law or agreed to in writing, software
+> distributed under the License is distributed on an "AS IS" BASIS,
+> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+> implied. See the License for the specific language governing
+> permissions and limitations under the License.
+
+
++++
+title = "IBM MQ"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on IBM MQ Queue"
+go_file = "ibmmq_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `ibmmq` trigger for IBM MQ Queue.
+
+```yaml
+triggers:
+    - type: ibmmq
+      metadata:
+        queueLength: '5' # OPTIONAL - Queue length target for HPA. Default: 5 messages
+        host: <ibm-host> # REQUIRED - IBM MQ Queue Manager Admin REST Endpoint
+        queueName: <queue-name> # REQUIRED - Queue Name
+      authenticationRef:
+        name: ibmmq-consumer-trigger
+```
+
+**Parameter list:**
+
+- `queueLength`: OPTIONAL - Queue Length Target for HPA. Will be set to Default Value of 5 if not Provided.
+- `host`: REQUIRED - IBM MQ Queue Manager Admin REST Endpoint. Example URI endpoint structure on IBM cloud `https://example.mq.appdomain.cloud/ibmmq/rest/v2/admin/action/qmgr/QM/mqsc`
+- `queueName`: REQUIRED - Name of the Queue within the Queue Manager defined from which messages will be consumed
+
+### Authentication Parameters
+
+TriggerAuthentication CRD is used to connect and authenticate to IBM MQ:
```

Comment:
Can you list the required parameters here as well please? We tend to do this for all scalers. See https://github.com/kedacore/keda-docs/blob/master/content/docs/2.0/scalers/azure-log-analytics.md#authentication-parameters

Reply:
Hi @tomkerkhove, we have listed the parameters as you have suggested in the latest commit